assets/app/view/game/corporation.rb

```diff
@@ -124,7 +124,8 @@ def render
       def render_title
         title_row_props = {
           style: {
-            grid: '1fr / auto auto',
+            grid: '1fr / repeat(auto-fit, auto)',
+            gridAutoFlow: 'column',
```

Comment:
Both lines have the same effect. In CSS later beats earlier, so the first line will be ignored:
![image](https://user-images.githubusercontent.com/33390595/105815032-998e7680-5fb2-11eb-8973-5c0fb27bcb88.png)